### ISSUE TITLE
Tiny mistake in the code

### ISSUE THREAD

**Issue by weisi-chen** [swcarpentry/r-novice-inflammation#245]
http://swcarpentry.github.io/r-novice-inflammation/04-cond/
In the Making Choices module, the very last command before the last exercise:

analyze_all("inflammation*.csv")

Here, "inflammation_.csv" is inconsistent with previous commands. It should be changed to "inflammation._csv". I suppose it takes a regex, so the .\* replaces for any number of characters after inflammation.

**Comment by jangari** [comment]
Markdown gobbled the asterisks. The point here is that `inflammation*.csv` is expanded to `inflammation` plus any number of `n`, plus any character, plus `csv`. 

This line should read `analyze_all("inflammation.*csv")`.

Eg:

```
> filenames1 <- list.files(path = "data", pattern = "inflammation*.csv")
> filenames1
character(0)
> filenames1 <- list.files(path = "data", pattern = "inflammation.*csv")
> filenames1
 [1] "inflammation-01.csv" "inflammation-02.csv" "inflammation-03.csv" "inflammation-04.csv" "inflammation-05.csv" "inflammation-06.csv" "inflammation-07.csv"
 [8] "inflammation-08.csv" "inflammation-09.csv" "inflammation-10.csv" "inflammation-11.csv" "inflammation-12.csv"
```

**Comment by RMHogervorst** [comment]
correct.
the period (.) means any character. the star after the period means repeat zero or more times.
So `.*`  means: [ any single character][matched zero or more times]

**Comment by bbolker** [comment]
I think this has been resolved?

**Comment by katrinleinweber** [comment]
Fixed by 3ee403989124343ffa8d47a08d54efb0ff8b38d5, but improvement discussion in https://github.com/swcarpentry/r-novice-inflammation/pull/337.
